player/lua.c

```diff
@@ -115,7 +114,6 @@ static int mp_cpcall (lua_State *L, lua_CFunction func, void *ud)
     lua_pushlightuserdata(L, ud);
     return lua_pcall(L, 1, 0, 0);
 }
-#define mp_lua_len lua_rawlen
```

Comment:
Why? In Lua 5.2 function lua_objlen was renamed lua_rawlen.

Reply:
fair, but will need to remember about it if anyone wants to use this.

Reply:
Even leaving it there you may not realize that define exists and still use `lua_objlen`. Neovim aliases `lua_rawlen` to `lua_objlen` instead at https://github.com/neovim/neovim/blob/66197dde7084484c9d23fa488b2288bcae364ba7/src/cjson/lua_cjson.c#L111 which prevents this.

Reply:
if it is unused, we can remove it